src/twisted/web/websocket.py

```diff
@@ -0,0 +1,32 @@
+# -*- test-case-name: twisted.web.test.test_websocket -*-
+"""
+Websocket (rfc6455) client and server support.
+
```

Comment:
Just some introduction to the main API.

```suggestion

Both client-side and server-side application code must implement the L{WebSocketProtocol} API.

```